### PR TITLE
[NTUSER] NtUserWaitForInputIdle: Fix unhandled exception BSOD 0x1E

### PR DESCRIPTION
**Pull request by TAN-Gaming**
## Purpose
Fix CLIENTINFO pointer access.
Based on a patch by Michael Fritscher from [CORE-10017](https://jira.reactos.org/browse/CORE-10017)
Addendum to 705228250741da (r68702).

JIRA issue:
[CORE-18728](https://jira.reactos.org/browse/CORE-18728)
[CORE-19014](https://jira.reactos.org/browse/CORE-19014) (duplicate)

### CODE REVIEW

File: win32ss/user/ntuser/message.c [hunk 1]
```diff
@@ -3176,12 +3176,17 @@ NtUserWaitForInputIdle( IN HANDLE hProcess,
     while (TRUE);
 
 WaitExit:
+    KeStackAttachProcess(&Process->Pcb, &ApcState);
+
```
Comment (JoachimHenze) [inline win32ss/user/ntuser/message.c:3180]:
```suggestion
```

File: win32ss/user/ntuser/message.c [hunk 2]
```diff
@@ -3176,12 +3176,17 @@ NtUserWaitForInputIdle( IN HANDLE hProcess,
...
     for (pti = W32Process->ptiList; pti; pti = pti->ptiSibling)
     {
        pti->TIF_flags &= ~TIF_WAITFORINPUTIDLE;
        pti->pClientInfo->dwTIFlags = pti->TIF_flags;
     }
     W32Process->W32PF_flags &= ~W32PF_WAITFORINPUTIDLE;
+
```
Comment (JoachimHenze) [inline win32ss/user/ntuser/message.c:3187]:
```suggestion
```

**Review by HBelusca — APPROVED** [review]
I'm curious to know whether there could be other places in win32k code where that stackattach/detachprocess call could be missing...

**Review by JoachimHenze — APPROVED** [review]

**Comment by Daniel7689** [comment]
This PR also fixes the issue mentioned at https://jira.reactos.org/browse/CORE-18728.

**Review by binarymaster — APPROVED** [review]

**Pull request merged**